ironfish/src/workerPool/tasks/decryptNotes.ts

```diff
@@ -26,6 +26,7 @@ export interface DecryptedNote {
 }
 
 export class DecryptNotesRequest extends WorkerMessage {
+  static nullValue = '0'.repeat(KEY_LENGTH)
```

Comment:
nit - I'm wondering if we should define a `NULL_KEY` somewhere since this gets used in a couple of places?

Reply:
Will delete this in subsequent migration PR